pom.xml

```diff
@@ -116,6 +118,9 @@
     <jersey.version>2.39.1</jersey.version>
     <jetty.version>9.4.52.v20230823</jetty.version>
     <jakarta.servlet-api.version>4.0.4</jakarta.servlet-api.version>
+    <!-- openapi-generator dependencies -->
+    <openapitools.jackson-databind-nullable.version>0.2.6</openapitools.jackson-databind-nullable.version>
+    <openapitools.swagger.version>1.6.11</openapitools.swagger.version>
```

Comment:
For maven, the openapi-generator version is 6.6.0, the latest version supports Java 8.
For sbt, the sbt-openapi-generator version is 5.0.1, the only version supports Java 8.

And due to openapi-generator 5.0.1 does not support <annotationLibrary>none</annotationLibrary> and <annotationLibrary>swagger2</annotationLibrary>,, so we have to use introduce swagger1 dependencies.
FYI:
https://openapi-generator.tech/docs/generators/java

We can remove swagger1.x dependencies after dropping the support for JDK8 and upgrade the openapi-generator to 7.x+ in the future.

Reply:
thanks for investigating, I agree the descision